server/gendb/interface.go.tmpl

```diff
@@ -36,8 +41,34 @@ type Interface interface {
     {{ end }}
 }
 
+// Int64 returns a pointer to the int64 value passed in.
+func Int64(i int64) *int64 { return &i }
+
+// String returns a pointer to the string value passed in.
+func String(s string) *string { return &s }
+
+{{- if anyPrimaryKeyUsesDateTime .XDBConfigs }}
+// DateTime returns a pointer to the strfmt.DateTime value passed in.
+func DateTime(d strfmt.DateTime) *strfmt.DateTime { return &d }
+{{- end }}
+
 {{- range $xdbConfig := .XDBConfigs }}
 {{- $modelName := pascalize $xdbConfig.SchemaName }}
+{{- $indexName := indexName $xdbConfig.DynamoDB.KeySchema }}
+{{- $hashKey := index $xdbConfig.DynamoDB.KeySchema 0 }}
+{{- $hashKeySchema := index $xdbConfig.Schema.Properties $hashKey.AttributeName }}
+{{- if indexHasRangeKey $xdbConfig.DynamoDB.KeySchema }}
+{{- $rangeKey := index $xdbConfig.DynamoDB.KeySchema 1 }}
+{{- $rangeKeySchema := index $xdbConfig.Schema.Properties $rangeKey.AttributeName }}
+// Get{{ $modelName }}sBy{{ $indexName }}Input is the query input to Get{{ $modelName }}sBy{{ $indexName }}.
+type Get{{ $modelName }}sBy{{ $indexName }}Input struct {
+	{{ pascalize $hashKey.AttributeName }} {{ goType $hashKeySchema }}
+        {{ pascalize $rangeKey.AttributeName }}StartingAfter *{{ goType $rangeKeySchema }}
```

Comment:
Am I understanding correctly that this is not always `StartingAfter`? Based on the `Descending` it would be before or after? Maybe `StartingFrom` is a better way to say this?

Reply:
It'll always be `After`. `Descending` controls the ordering of the results returned, not the traversal of the index

Reply:
I do like the name `From` better though, since the constraint is `>=`. Will change it to that

Reply:
renamed to `...StartingAt`: 183c484